azure-pipelines.yml

```diff
@@ -0,0 +1,117 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
+
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python35:
+        python.version: '3.5'
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 4
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+   
+  - script: |
+      pip install --upgrade pip
+      pip install pytest
+      pip install -U cookiecutter
+      cookiecutter --no-input https://github.com/Azure/cookiecutter-azure-iot-edge-module module_name=filterModule image_repository=localhost:5000/filtermodule --checkout master
```

Comment:
What's the purpose of this CI pipeline? For daily test, or for CI? Anyway, we should reference the branch we want to test here, instead of hard coding `master` here.

Reply:
It's for daily test so that hard coding master here is better.

Reply:
@neil-yechenwei We just need enable CI for each commit and PR, not daily test. For now, we are only testing the content of the template. No need to do nightly run. Please update to test againt the the source code not master branch. In the future, if we have integration test (e.g. test `docker build` with the template), we may wanto enable daily test.